streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/ClientStateTest.java

```diff
@@ -406,12 +428,31 @@ public void shouldAddTasksInOffsetSumsMapToPrevStandbyTasks() {
             mkEntry(TASK_0_2, 100L)
         );
         client.addPreviousTasksAndOffsetSums("c1", taskOffsetSums);
-        client.initializePrevTasks(Collections.emptyMap());
+        client.initializePrevTasks(Collections.emptyMap(), false);
         assertThat(client.prevStandbyTasks(), equalTo(mkSet(TASK_0_1, TASK_0_2)));
         assertThat(client.previousAssignedTasks(), equalTo(mkSet(TASK_0_1, TASK_0_2)));
         assertTrue(client.prevActiveTasks().isEmpty());
     }
 
+    @Test
+    public void shouldThrowWhenSomeOwnedPartitionsAreNotRecognizedWhenInitializingPrevStandbyTasks() {
```

Comment:
These two tests seem only be different on the test names, but they are the same as the ones above (all lines within the function are the same)?

Reply:
Ah, whoops, yeah they used to differ but I had to take that line out -- I'll remove the duplicate tests